tests/index.test.js

```diff
@@ -300,7 +309,7 @@ describe('index', () => {
       expect(fs.readdir).toHaveBeenNthCalledWith(1, src);
       expect(fs.readdir).toHaveBeenNthCalledWith(2, path.join(src, 'utils'));
       expect(fs.stat).toHaveBeenCalledTimes(
-        filesRoot.length + filesUtils.length - 1, // .eslintrc
+        filesRoot.length + filesUtils.length - 3, // . .. node_modules
```

Comment:
You added `.`, `..`, `node_modules` and an extra `isDirectory: () => false`, but `findFiles` filters them before calling `fs.stat`: `!['.', '..', 'node_modules'].includes(item)`, so, why extra mock, and the change in here?